lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -90,6 +92,7 @@ def refund(money, authorization, options = {})
         add_reference(post, authorization, options)
         add_splits(post, options)
         add_network_transaction_reference(post, options)
+        add_recurring_detail_reference(post, options)
```

Comment:
This endpoint doesn't seem to support this field, https://docs.adyen.com/api-explorer/Payment/68/post/refund

Reply:
Done!